src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Intervention.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.JoinTable
+import javax.persistence.ManyToMany
+import javax.persistence.ManyToOne
+
+enum class AgeGroup {
+  EIGHTEEN_TO_25, OVER_25
+}
+
+@Entity
+data class Intervention(
+  @Id @GeneratedValue val id: UUID? = null,
+  @CreationTimestamp val createdAt: OffsetDateTime? = null,
+
+  val title: String? = null,
+  val description: String? = null,
+
+  @Enumerated(EnumType.STRING) var ageGroup: AgeGroup? = null,
+
+  @ManyToOne @JoinColumn(name = "service_category_id")
+  var serviceCategory: ServiceCategory? = null,
+
+  @ManyToMany
+  @JoinTable(
+    name = "intervention_region",
+    joinColumns = [JoinColumn(name = "intervention_id")],
+    inverseJoinColumns = [JoinColumn(name = "region_id") ]
+  )
+  var regions: List<Region>? = emptyList()
```

Comment:
since regions is unordered, it seems like (according to the post below) using a `Set` may produce more efficient sql for this field

https://vladmihalcea.com/the-best-way-to-use-the-manytomany-annotation-with-jpa-and-hibernate/